markdown/org/docs/patterns/lucy/cutting/en.md

```diff
@@ -4,3 +4,15 @@ title: "Lucy tie-on pocket: Cutting Instructions"
 
 - Cut **2 pocket** parts
 - Cut pocket slit into **1 pocket** part
+
+#### Historical context 
+
+Tie-on pockets like Lucy were worn roughly from the mid-seventeenth (1650s) century until the end of the 19th century (1890s). They came in all shapes and sizes but Lucy is based on an 18th century example. 
+
+They were worn and used by all members of society and were mainly womens wear as men had a variety of pockets sewn into their clothing. 
+
+Pockets would be tied around the waist and could be accessed through a slit in skirts and petticoats. Where inbetween the layers the pocket would be was up to the individual. If it was hidden in between layers that would provide more security, but hinder accessibility. People who worked in trade often wore their pockets on the outside of their garments entirely, for easy access. 
+
+(Tie-on) Pockets were used alongside patch pockets, sewn in pockets, bags and other ways to carry your things. Sometimes they even had additional pockets hidden inside them! 
+
+If you want to learn more you can read: Burman, B. & Fennetaux A. (2020)  _The Pocket, A Hidden History of Women's Lives_ Yale University Press, London
```

Comment:
I'm going to tick ths off but I am thinking it may be nice to link the book but since it is a product that needs to be brought rather than a free resource I felt it would be best not to link and let those who are more inquistive do the research to find it.

Reply:
Ive deliberately added the book in such a way that it should be easy to find, whether you want to buy it or check it out in the library :)

Reply:
I would suggest changing the "if you want to learn more you can read:" wording, as the tone can come off as slightly promotional. Instead, I would use something simple like "Reference:" to make it more of a neutral citation.

Reply:
I don't want to do that, because this isn't really an academic article.  I mean it very literally, if people want to learn more I do recommend the book I referenced because it's the seminal text on the topic. I can convert it into a note with my own name on it if that seems better.